app/templates/components/events/view/export/api-response.hbs

```diff
@@ -7,36 +7,36 @@
 </div>
 <div class="ui form">
   <div class="field">
-    {{ui-checkbox class='toggle' label=(t 'Basic Event Details')}}
+    {{ui-checkbox class='toggle' label=(t 'Basic Event Details') checked=toggleSwitches.basicEventDetails onChange=(action "checkboxChange" 'basicEventDetails')}}
   </div>
   <div class="field">
-    {{ui-checkbox class='toggle' label=(t 'Sessions')}}
+    {{ui-checkbox class='toggle' label=(t 'Sessions') checked=toggleSwitches.sessions onChange=(action "checkboxChange" 'sessions')}}
```

Comment:
Use single-quotes within handlebar syntax. For uniformity. And use double-quotes for HTML